android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetOngoingConferenceService.java

```diff
@@ -132,10 +154,16 @@ public void onCreate() {
             stopSelf();
             JitsiMeetLogger.w(TAG + " Couldn't start service, notification is null");
         } else {
+            if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.Q) {
```

Comment:
This and the next check both cover Q. Which one should Q execute? Also make it if-else if-else please.